File: progra3-0583-equipo5-frontend/Frontend_C#/SoftPac/SoftPacBusiness/EntidadesBancariasBO.cs
using SoftPac.Model;
using SoftPac.Persistance.DAO;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftPac.Business
{
    public class EntidadesBancariasBO
    {
        private EntidadesBancariasDAO entidadesDAO;

        public EntidadesBancariasBO()
        {
            this.entidadesDAO = new EntidadesBancariasDAOImpl();
        }

        public IList<EntidadesBancariasDTO> ListarTodos()
        {
            return entidadesDAO.ListarTodos();
        }

        public EntidadesBancariasDTO ObtenerPorId(int bancoId)
        {
            return entidadesDAO.ObtenerPorId(bancoId);
        }

    }
}

File: BackEnd/Backend_Java/SoftPacBusiness/src/main/java/pe/edu/pucp/softpac/bo/EntidadesBancariasBO.java
package pe.edu.pucp.softpac.bo;

import java.util.ArrayList;
import pe.edu.pucp.softpac.dao.EntidadesBancariasDAO;
import pe.edu.pucp.softpac.daoImpl.EntidadesBancariasDAOImpl;
import pe.edu.pucp.softpac.model.EntidadesBancariasDTO;

public class EntidadesBancariasBO {
    private EntidadesBancariasDAO entidadesBancariasDAO;
    
    public EntidadesBancariasBO(){
        entidadesBancariasDAO = new EntidadesBancariasDAOImpl();
    }
    
    public ArrayList<EntidadesBancariasDTO> listarTodos(){
        return (ArrayList<EntidadesBancariasDTO>) this.entidadesBancariasDAO.listarTodos();
    }
    
    public EntidadesBancariasDTO obtenerPorId(Integer entidades_bancarias_id){
        EntidadesBancariasDTO entidadesBancariasDTO = new EntidadesBancariasDTO();
        entidadesBancariasDTO.setEntidad_bancaria_id(entidades_bancarias_id);
        return this.entidadesBancariasDAO.obtenerPorId(entidades_bancarias_id);
    }
    
}